instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py

```diff
@@ -368,6 +489,27 @@ def test_requests_exception_without_response(self, *_, **__):
         )
         self.assertEqual(span.status.status_code, StatusCode.ERROR)
 
+    @mock.patch(
+        "requests.adapters.HTTPAdapter.send",
+        side_effect=requests.RequestException,
+    )
+    def test_requests_exception_new_semconv(self, *_, **__):
+        with self.assertRaises(requests.RequestException):
+            self.perform_request(self.URL)
+
+        span = self.assert_span()
+        self.assertEqual(
+            span.attributes,
+            {
+                SpanAttributes.HTTP_REQUEST_METHOD: "GET",
+                SpanAttributes.URL_FULL: self.URL,
+                SpanAttributes.SERVER_ADDRESS: self.HOST,
+                _SPAN_ATTRIBUTES_NETWORK_PEER_ADDRESS: "mock",
+                _SPAN_ATTRIBUTES_ERROR_TYPE: "RequestException",
```

Comment:
exception type should be fully qualified

Reply:
Changed to `__qualname__` in code.